src/EnergyPlus/MixedAir.cc

```diff
@@ -3715,63 +3670,37 @@ Real64 VentilationMechanicalProps::CalcMechVentController(EnergyPlusData &state,
         } else {
             // for system OA methods: Zone_Sum, VRP, CO2 methods
             // new code for DCV method complying with the VRP defined in ASHRAE Standard 62.1-2010
+            bool const useOccSch = this->DCVFlag && (this->SystemOAMethod != DataSizing::SysOAMethod::ProportionalControlDesOcc);
 
             // Loop through each zone first to calc uncorrected system OA flow rate
-            SysOAuc = 0.0;
-            SysOA = 0.0;
+            SysOAuc = 0.0; // [m3/s]
+            SysOA = 0.0;   // [m3/s]
             for (int ZoneIndex = 1; ZoneIndex <= this->NumofVentMechZones; ++ZoneIndex) {
```

Comment:
In `CalcMechVentController` replace blocks code to calculate OA flow rates . . .